src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -138,7 +139,7 @@ export class MeetingManager implements AudioVideoObserver {
   // Please use `meetingManagerConfig` to use `MeetingManagerConfig` values.
   postLoggerConfig: PostLogConfig | null = null;
 
-  eventReporter: EventReporter;
+  eventController: EventController;
```

Comment:
We have not used encapsulation at all in component library, so moving forward can we make this a read-only private param or have getters? I think we do not want to override this directly using `meetingManager` right?

Reply:
Where else would we override it? I'm not too familiar with our react setup, so there could be a better place that I'm not aware of. However this was how we used to override the EventReporter so ideally I think we should try and keep it similar.